### PR TITLE
Use Locale.ROOT when user toUpperCase or toLowerCase

### LINKED ISSUE #6048: Use LocaleRoot when using Strings toLowerCase/toUpperCase
> [!WARNING]
> Do not convert this to an issue until you remove the notes on the CVE

Update usage for String toLowerCase / toUpperCase to use the Locale.ROOT.

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #6048

### CODE REVIEW

File: spring-cloud-skipper/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/PostgreSQLTextToOID.java [hunk 1]
```diff
@@ -52,7 +53,7 @@ public class PostgreSQLTextToOID {
 	public static void convertColumnToOID(String table, String id, String column, DataSource dataSource) {
 
 		try (Connection connection = dataSource.getConnection()) {
-			try (ResultSet resultSet = connection.getMetaData().getColumns(null, null, table.toUpperCase(), column.toUpperCase())) {
+			try (ResultSet resultSet = connection.getMetaData().getColumns(null, null, table.toUpperCase(Locale.ROOT), column.toUpperCase())) {
```
Comment (corneil) [inline spring-cloud-skipper/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/PostgreSQLTextToOID.java:56]: You missed the colum.toUpperCase()

File: spring-cloud-skipper/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/PostgreSQLTextToOID.java [hunk 2]
```diff
@@ -90,7 +91,7 @@ public static void convertColumnToOID(String table, String id, String column, Da
 
 	public static void convertColumnFromOID(String table, String id, String column, DataSource dataSource) {
 		try (Connection connection = dataSource.getConnection()) {
-			try (ResultSet resultSet = connection.getMetaData().getColumns(null, null, table.toUpperCase(), column.toUpperCase())) {
+			try (ResultSet resultSet = connection.getMetaData().getColumns(null, null, table.toUpperCase(), column.toUpperCase(Locale.ROOT))) {
```
Comment (corneil) [inline spring-cloud-skipper/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/PostgreSQLTextToOID.java:94]: You missed `table.toUpperCase()`

Comment (cppwfs) [reply spring-cloud-skipper/spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/db/migration/PostgreSQLTextToOID.java:94]: WOOT WOOT!   Great Catches!

**Review by corneil — APPROVED** [review]
:100:

**Pull request merged**